docs/supported-libraries.md

```diff
@@ -214,10 +214,10 @@ These are the application servers that the smoke tests are run against:
 
 These are the JVMs and operating systems that the integration tests are run against:
 
-| JVM                                                                                       | Versions          | OS                                    |
-| ----------------------------------------------------------------------------------------- |-------------------| ------------------------------------- |
-| [OpenJDK (Eclipse Temurin)](https://adoptium.net/)                                        | 8, 11, 17, 21, 22 | [`ubuntu-latest`], [`windows-latest`] |
-| [OpenJ9 (IBM Semeru Runtimes)](https://developer.ibm.com/languages/java/semeru-runtimes/) | 8, 11, 17, 21     | [`ubuntu-latest`]                     |
+| JVM                                                                                       | Versions          | OS                |
+|-------------------------------------------------------------------------------------------|-------------------|-------------------|
+| [OpenJDK (Eclipse Temurin)](https://adoptium.net/)                                        | 8, 11, 17, 21, 23 | [`ubuntu-latest`] |
+| [OpenJ9 (IBM Semeru Runtimes)](https://developer.ibm.com/languages/java/semeru-runtimes/) | 8, 11, 17, 21, 23 | [`ubuntu-latest`] |
```

Comment:
currently we don't run tests on openj9 23 https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d3e1a2c504d6fab0e4f77c486f3aebed0ee27647/.github/workflows/build-common.yml#L210-L213 I think this is because openj9 23 wasn't available when we update to openj9 23.

Reply:
thanks! fixed